save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/OrganizationController.kt

```diff
@@ -341,20 +342,32 @@ internal class OrganizationController(
             "Could not find an organization with name $organizationName."
         }
         .filter {
-            organizationPermissionEvaluator.hasPermission(authentication, it, Permission.DELETE)
+            organizationPermissionEvaluator.hasPermissionToChangeStatus(authentication, it, status)
         }
         .switchIfEmptyToResponseException(HttpStatus.FORBIDDEN) {
-            "Not enough permission for deletion of organization $organizationName."
+            "Not enough permission for this action with organization of organization $organizationName."
         }
         .filter {
-            !organizationService.hasProjects(organizationName)
+            status == OrganizationStatus.DELETED && !organizationService.hasProjects(organizationName)
```

Comment:
So if `status` is not `OrganizationStatus.DELETED` user will get message `There are projects connected to $organizationName. Please delete all of them and try again"`? Is it what was planned or did I miss something?

Reply:
Yes, thanks, I changed it to
``status != OrganizationStatus.DELETED || !organizationService.hasProjects(organizationName)```